perun-registrar-lib/src/test/java/cz/metacentrum/perun/registrar/AppAutoRejectionSchedulerTest.java

```diff
@@ -180,11 +180,6 @@ private void setUpVo() throws Exception {
 	 * @throws GroupExistsException if group already exists
 	 */
 	private Group deleteApplicationFormAndCreateGroup() throws GroupExistsException {
```

Comment:
We should probably rename this method and update javadoc, since its no longer deleting application form to fix weird behavior of HSQLDB.

Reply:
Done